res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" ?>
+<MixxxKeyboardPreset mixxxVersion="2.0.1+" schemaVersion="1">
+  <info>
+    <name>Default Mapping</name>
+    <author>Mixxx</author>
+    <description>Default mapping for Mixxx.</description>
+  </info>
+  <controller>
+    <group name="[AutoDJ]">
+      <control action="enabled">
+        <keyseq final="1">Shift+F12</keyseq>
+      </control>
+      <control action="fade_now">
+        <keyseq final="1">Shift+F11</keyseq>
+      </control>
+      <control action="shuffle_playlist">
+        <keyseq final="1">Shift+F9</keyseq>
+      </control>
+      <control action="skip_next">
+        <keyseq final="1">Shift+F10</keyseq>
+      </control>
+    </group>
+    <group name="[Channel1]">
+      <control action="LoadSelectedTrack">
+        <keyseq final="1">Shift+Left</keyseq>
+      </control>
+      <control action="back">
+        <keyseq lang="en_US" scancode="30">a</keyseq>
```

Comment:
Yeah! It's done :) Both the `kbdcfg_to_kbdxml` tool as the parser are updated to support this new format. It took a while because we now moved the scancode retrieval from the `kbdcfg_to_kbdxml` tool to the Mixxx C++ domain.

Oh, and one little funky thing that is also possible with this format:
```XML
<!-- Binds to whatever key that has the scancode of the 
'a' labeled key on "en_US", on the "ru_RU" layout. -->

<keyseq byPositionOf="en_US" lang="ru_RU">a</keyseq> 
```